src/commands/perform-tasks.ts

```diff
@@ -70,6 +72,11 @@ export class PerformTasksCommand extends BaseCliCommand<IPerformTasksCommandArgs
         const tasks = config.enumBuildTasks(command);
         ConsoleUtil.state = state;
 
+        if(command.taskMatcher) {
+            const tasksPrefix = '';
+            this.skipNonMatchingLeafTasks(tasks, command.taskMatcher, tasksPrefix);
```

Comment:
```suggestion
            const num = this.skipNonMatchingLeafTasks(tasks, command.taskMatcher, '');
            if (num === 0) {
                ConsoleUtil.LogWarning(`--match parameter ${command.taskMatcher} did not match any tasks`);
            }
```

Reply:
what do you think about this? to give better feedback if a `--match` parameter was provided but it did not match any task?

Reply:
done